src/init.cpp

```diff
@@ -1031,6 +1031,19 @@ bool AppInitParameterInteraction(const ArgsManager& args, bool use_syscall_sandb
         return InitError(_("No proxy server specified. Use -proxy=<ip> or -proxy=<ip:port>."));
     }
 
+    if (args.GetBoolArg("-reindex-chainstate", false)) {
+        // indexes that must be deactivated to prevent index corruption, see #24630
```

Comment:
not sure how helpful it is to write `#24630`. Either this should be a full URL or even better a description for devs to explain which indexes must be deactivated and which ones do not.

If all need to be deactivated it would be better to write "All index" instead of "indexes that".